external/libtww/d/save/d_save.h

```diff
@@ -281,7 +288,11 @@ static_assert(sizeof(dSv_player_get_bag_item_c) == 0xC);
 
 class dSv_player_bag_item_record_c {
 public:
-    /* 0x00 */ u8 field_0x0[8];
+    u8 getSpoilsNum(u8 index) { return mSpoilsBagCounts[index]; }
+
+    void setSpoilsNum(u8 index, u8 amount) { mSpoilsBagCounts[index] = amount; }
+
+    /* 0x00 */ u8 mSpoilsBagCounts[8];
     /* 0x08 */ u8 field_0x8[8];
     /* 0x10 */ u8 field_0x10[8];

```

Comment:
I'm guessing these unknown fields are "mBaitBagCounts" and "mDeliveryBagCounts".  Is there a reason they are not being used?

Reply:
you cant stack delivery bag items but i could add it anyway